chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceNonconformity.java

```diff
@@ -90,7 +91,7 @@
      * Date non-conformity was closed
      */
     @XmlElement(required = false, nillable = true)
-    private Date nonConformityCloseDate;
+    private LocalDate nonconformityCloseDate;
```

Comment:
This is more a question than anything, but the RWT Date fields (and all the LocalDate fields I know of that were copied from those original ones) have these annotations. Does this field need them too?
`   @JsonDeserialize(using = LocalDateDeserializer.class)
    @JsonSerialize(using = LocalDateSerializer.class)
    @XmlJavaTypeAdapter(value = LocalDateAdapter.class)`